packages/lib/src/components/Table/TableControl/TableControl.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import { CustomStoryObj, CustomMeta } from "test_utils/types";
```

Comment:
When creating new stories you need to run `yarn snapshot -u` to regenerate storyshots.